samples/system-test/hmacKey.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {Storage} = require(`@google-cloud/storage`);
+const {assert} = require('chai');
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const poolProjectId = process.env.POOL_PROJECT_ID;
```

Comment:
These are new!  Is there work that needs to be done to make sure these are piped through to the kokoro tests?

Reply:
@jkwlui is working on introducing a pool to test these samples and client system tests as well.